humanizer/operation.py

```diff
@@ -120,8 +120,12 @@ def add_event(self, rest):
         tokenized_rest = rest.split(" ")
 
         if tokenized_rest[0] in ["BIND", "SRCH", "EXT", "STARTTLS", "UNBIND", "CMP", "WHOAMI"]:
-            self.requests.append(
-                {"verb": tokenized_rest[0], "details": tokenized_rest[1:]})
+            if len(self.requests) > 0:
+                if tokenized_rest[0] == self.requests[0]["verb"]:
+                    self.requests[0]["details"].extend(tokenized_rest[1:])
```

Comment:
hard coding the request ID here will result in bugs when we see something like this...

    Oct 26 03:40:04 ldap.example.com slapd[11086]: conn=6834744 op=9 EXT oid=1.3.6.1.4.1.4203.1.11.3
    Oct 26 03:40:04 ldap.example.com slapd[11086]: conn=6834744 op=9 BIND foo
    Oct 26 03:40:04 ldap.example.com slapd[11086]: conn=6834744 op=9 BIND bar

I think we need to just iterate over the existing operations and append if we need an exact op match and if not, then create new.